### PR TITLE
LPD-15524 Pages portlet becomes unavailable after a modified page in publication is deleted from production

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
@david-truong can you review my changes?

**Comment by pei-jung** [comment]
ci:test:relevant

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -10377,6 +10377,19 @@
...
 
+##
+## Change Tracking
+##
+
+    #
+    # Set this to true to enable deletion protection for publication changes. If
+    # enabled, an object cannot be deleted in production if it's being modified
+    # in a publication.
+    #
```
Comment (david-truong) [inline portal-impl/src/portal.properties:10388]: can you spell out change tracking instead of using ct?

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:report:2532277000

**Pull request closed without merging**